public/app/features/alerting/unified/components/rule-editor/alert-rule-form/ModifyExportRuleForm.tsx

```diff
@@ -28,28 +39,41 @@ export function ModifyExportRuleForm({ ruleForm, alertUid }: ModifyExportRuleFor
     defaultValues: ruleForm,
     shouldFocusError: true,
   });
+  const [queryParams] = useQueryParams();
```

Comment:
You can consider using `useURLSearchParams` as it gives access to the built-in `URLSearchParams` object that allows you to easily retrieve query params without thinking about types conversions and multiple parameters.
You could use it like `queryParams.get('returnTo')`